gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -740,6 +740,11 @@ gst_tensor_transform_set_option_data (GstTensorTransform * filter)
   if (filter->mode == GTT_UNKNOWN || filter->option == NULL)
     return TRUE;
 
+  if (strlen (filter->option) == 0) {
+    ml_loge ("Given option string is empty, you should set proper option.");
+    return FALSE;
```

Comment:
If ```filter->option == NULL``` is fine, can we consider ```filter->option == ""``` equally with NULL?

Reply:
Default value of filter->option is null, which means it is not parsed yet in this line. (or..developer does not set an option string.

Reply:
@jaeyun-jung Is it necessary? IMO, ```g_regex_match_simple()``` handles the case where ```strlen(filter->option)``` is ```0```?

Reply:
right, it is better to add new regex for stand mode. (currently no pattern for stand mode)

Reply:
@jaeyun-jung Understood. After merging this PR, I will add ```g_regex_match_simple()``` for the stand mode (and others if any). Thanks.

Reply:
@wooksong plz consider new option for dc-offset removal (#3129)